doc/examples/edges/plot_circular_elliptical_hough_transform.py

```diff
@@ -82,10 +82,10 @@
 # -------------------
 #
 # The algorithm takes two different points belonging to the ellipse. It
-# assumes that it is the main axis. A loop on all the other points determines
-# how much an ellipse passes to them. A good match corresponds to high
-# accumulator values.
-#
+# assumes that it is the major axis. A loop on all the other points determines
+# potential minor axis lengths of candidate ellipses. Ellipses are added to the
+# results if enough valid candidates have similar minor axis lengths.
+# 
```

Comment:
```suggestion
#
```
(I understand the whitespace was probably added by your text editor.)